src/shared/settingsConfiguration.ts

```diff
@@ -56,4 +56,50 @@ export class DefaultSettingsConfiguration implements SettingsConfiguration {
             return false
         }
     }
+    /**
+     * Sets a prompt message as suppressed.
+     * @param promptName Name of prompt to append to list
+     */
+    public disable(promptName: string): void {
+        const setting = this.getSuppressPromptSetting(promptName)
+        if (setting === undefined || setting[promptName]) {
+            return
+        }
+        setting[promptName] = true
+        this.writeSetting('suppressPrompts', setting, vscode.ConfigurationTarget.Global)
+    }
+    /**
+     * Verifies if a prompt should be displayed again.
+     * @param promptName Name of the prompt
+     * @returns False when prompt has been suppressed
+     */
+    public shouldDisplayPrompt(promptName: string): boolean {
+        const promptSetting = this.getSuppressPromptSetting(promptName)
+        if (promptSetting !== undefined && promptSetting[promptName]) {
+            return false
+        }
+        return true
+    }
+
+    /**
+     * Gets the 'suppressPrompts' settings object. This will reset the setting to default if it does not
```

Comment:
fully-qualified name is less indirection at little cost.
```suggestion
     * Gets the 'aws.suppressPrompts' setting. This will reset the setting to default if it does not
```